docs/content-api.md

```
@@ -0,0 +1,137 @@
+    "type": "media--civictheme_image",
+    "attributes": { "name": "Descriptive name" },
+    "relationships": {
+      "field_c_m_image": {
+        "data": { "type": "file--file", "id": "<file-uuid>", "meta": { "alt": "Describe the image" } }
+      }
+    }
+  }
+}
+```
+
+3. Reference the media's `id` from a component's image field (e.g. `field_c_p_image`).
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/1/1

(MD029, ol-prefix)